.github/scripts/generate-os-packages.sh

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env bash
 set -eu
 
+ARCHITECTURE="x86_64" # Set the default architecture
+if [[ "$OSTYPE" == "linux-gnu"* ]] || [[ "$OSTYPE" == "darwin"* ]]; then
+  # When running on macOS and Linux lets determine the architecture
+  ARCHITECTURE=$(uname -m)
+fi
+if [[ $# -eq 1 ]]; then
+  # architecture gets overridden by command line param
+  ARCHITECTURE=$1
+fi
```

Comment:
This is not really necessary anymore, but I still kept it, so in future, if possible with Graal, you can pass the architecture as first argument for cross building. Currently it has no use.

Reply:
It's fine for me. We can still keep it here.